ww-config.js

```diff
@@ -5,6 +5,9 @@ export default {
         },
         excludedSections: ["sizing", "spacing", "positioning", "background", "styling", "advanced-style", "customCss"],
     },
+    triggerEvents: [
+        { name: 'backdropClick', label: { en: 'On backdrop click' }, event: { value: '' } },

```

Comment:
do we really want to emit a value?

Reply:
Not necessary I think

Reply:
Mmh j'ai cru que c'était une convention qu'on avait, si on emit rien on met un objet vide ou null ? On veut que ça s'affiche dans la partie workflow de l'explorer même si il y a rien ? @Kramerbaf @AurelieV

Reply:
Si c'est vide rien ne doit s'afficher dans l'explorer non